Makefile

```diff
@@ -88,16 +87,22 @@ do-pre-init:
 	cp -n config/config.example.json config/config.json
 
 do-switch-branch:
-	@if [ -z $$BRANCH ]; then echo "No branch is set, please run:\nmake update BRANCH=<branch>"; exit 1; fi
-	@echo "\n=== Switching to and updating $$BRANCH ===\n"
-	git checkout $$BRANCH
-	git pull upstream $$BRANCH
+	@if [ -z $$BRANCH ]; then \
+	    echo "\n=== Running updates for the current branch ===\n"; \
+	    echo "No branch is set, run: 'make update BRANCH=<branch>' to checkout and update a branch"; \
+	else \
+	    echo "\n=== Switching to and updating $$BRANCH ===\n"; \
+	    git checkout $$BRANCH; \
+	    git pull upstream $$BRANCH; \
+	fi
 
 do-checkout-pr:
-	@if [ -z $$PR ]; then echo "No PR number is set, please run:\nmake github PR=<number>"; exit 1; fi
-	@echo "\n=== Checking out Pull Request $$PR ===\n"
-	git fetch upstream refs/pull/$$PR/head:refs/remotes/upstream/pr/$$PR
-	git checkout upstream/pr/$$PR
+	@echo "\n=== GitLab Merge Request ===\n"
```

Comment:
Please update this to GitHub Pull Request :)

Reply:
Fixed! Lame :smile: